libevmasm/Assembly.cpp

```diff
@@ -681,6 +681,9 @@ LinkerObject const& Assembly::assemble() const
 			ret.bytecode.push_back(uint8_t(Instruction::PUSH20));
 			ret.bytecode.resize(ret.bytecode.size() + 20);
 			break;
+		case AccountAbstractionPrefix:
+			ret.bytecode.insert(ret.bytecode.end(), aaPrefixBytecode, aaPrefixBytecode+sizeof(aaPrefixBytecode));
```

Comment:
_technically_ you need to use `sizeof(aaPrefixBytecode) / sizeof(uint8_t)` because C++ is weird.

Reply:
I added a dedicated `const size_t aaPrefixLength` (as initially proposed by you), I think that is the cleanest solution